docs/topics/i18n/translation.txt

```diff
@@ -1373,10 +1373,16 @@ function. Example::
 
 .. warning::
 
-    :func:`~django.conf.urls.i18n.i18n_patterns` is only allowed in your root
+    :func:`~django.conf.urls.i18n.i18n_patterns` is only allowed in a root
     URLconf. Using it within an included URLconf will throw an
     :exc:`~django.core.exceptions.ImproperlyConfigured` exception.
 
+.. versionchanged:: 1.10
+
+    Using ``i18n_patterns`` in a root URLconf different from
```

Comment:
I think this can be all removed as theres some documentation about it in [HttpRequest](https://docs.djangoproject.com/en/1.9/ref/request-response/#django.http.HttpRequest.urlconf).